source/release/4.3.8/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: oVirt 4.3.8 Release Notes
+category: documentation
+layout: toc
+authors: Lev Veyde
+---
+
+<style>
```

Comment:
this is making our lives complicated. I saw various layers of overrides at different levels when migrating to Jekyll, made some cleanups but I could not disentangle everything, so please stop doing this. If you need changes of the style, then create a style for releases for example.

Reply:
@duck-rh let's do this in a batch, replacing all style for releases once.

Reply:
I see it's tracked in https://github.com/oVirt/ovirt-site/issues/2154